src/Fluid/Flu_ResetByUser.cpp

```diff
@@ -291,7 +291,7 @@ void Flu_ResetByUser_API_Default( const int lv, const int FluSg, const int MagSg
 //       2-2. operations necessary only when this cell has been reset
          if ( Reset )
          {
-#           if ( MODEL == HYDRO )
+#           if ( MODEL == HYDRO && !defined SRHD )
```

Comment:
Please also update the comment at line 313 to be `# endif // #if ( MODEL == HYDRO  &&  !defined SRHD )`, and there should be two spaces around && to match the code style.